routing/cross_mwm_index_graph.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "routing/segment.hpp"
+
+#include <vector>
+
+namespace routing
+{
+/// \brief This is an interface for cross mwm routing.
+// @TODO(bykoianko) This interface will have two implementations.
+// * For orsm cross mwm section.
+// * For A* cross mwm section
+class CrossMwmIndexGraph
+{
+public:
+  /// \returns true if |s| is a transition segment. That means the segment lies on
+  /// mwm border and it's possible to leave the mwm of |s| moving by the segment |s|.
+  virtual bool IsBorder(Segment const & s) const = 0;
```

Comment:
Could you please explain what the method should return in the following cases:
* start point of the segment lies on the border, end point doesn't
* start point of the segment doesn't lie on the border, end point does
* both points lie on the border
* both points do not lie on the border